### ISSUE TITLE
Implement total stars badge among all years

### ISSUE THREAD

**Issue by obalunenko** [joblo2213/AoC-badges#10]
I have all years in one repo and it will be nice to have the total stars collected among all years under one badge.

You can see it on this page: https://adventofcode.com/events

`Total stars:  n*`

Default regex for this badge could be:

```
'(?<=https:\/\/img\.shields\.io\/badge\/total%20stars%20⭐-)[0-9]+(?=-yellow)'
```

**Comment by J0B10** [comment]
Thanks for the suggestion, such a badge would be cool 😄 

Implementing this would require an additional request to https://adventofcode.com/events and parsing the total stars from the returned webpage.  

This is problematic for two reasons:  
* More requests to adventofcode.com (they should be limited to once every 15 minutes (900 seconds))
* It's not officially supported api, the html webpage must be parsed (but it could change at any time if Eric decides to).

I'll have to think about this.

**Comment by obalunenko** [comment]
@J0B10 Thanks for answering.

I just checked, https://adventofcode.com/2021/events (and even https://adventofcode.com/2020/events) returns the same content as https://adventofcode.com/events
So maybe it will be not needed to make an additional request.

**Comment by J0B10** [comment]
Currently the only requests are against the api for your private leaderboard. So an additional request against the events page would be needed.

**Comment by obalunenko** [comment]
Ah, exactly - now I see

**Comment by J0B10** [comment]
I thought about this and decided not to implement this. To sum up my reasons:

* I would have to make two separate requests to aoc sites (events & leaderboard) whenever the actin is run. Altough it should not add significant load it does still violate Eriks rule of only 1 request per 15min.  I will strictly adhere to this rule.
* I don't want to maintain code relying on non api sites. Altough Erik hasn't changed this site for years there is no guarantee that this will be true for the future. 
* If I start adding funtionallity like this it's unclear where to draw the line: Should I also add a global leaderboard score?  
  That would be another interesting data for a badge but require another request and parsing another html site. 

To sum up I will build this action only on data that is available from the json api.

Parsing html sites to fetch additional interesting stats for badges could be made into a separate action in the future.

**Comment by J0B10** [comment]
Maybe https://github.com/stackcats/adventofcode-badge does provide a solution for you.

**Comment by obalunenko** [comment]
Thanks for sharing your decision with fare reasons and I appreciate for providing solution for me 🙏🏿
